web/client/epics/details.js

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright 2020, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { Observable } from 'rxjs';
```

Comment:
hi, would be possible to add some unit test here?

in general new code for epics and reducer should be covered by unit tests at least for the most important pieces.

Not sure if know it, but when you run npm test or npm run continuoustest, a coverage report is produced in coverage/indedx.html 
i suggest you to have a look at it and examine how your changes re covered by the tests.
if you want to know more contact me 😺